.vscode/extensions.json

```diff
@@ -0,0 +1,10 @@
+{
+  "recommendations": [
+    "bierner.github-markdown-preview",
+    "bierner.jsdoc-markdown-highlighting",
+    "dbaeumer.vscode-eslint",
+    "ghmcadams.lintlens",
+    "rvest.vs-code-prettier-eslint",
+    "sidneys1.gitconfig"
```

Comment:
Should we add extension for `html` and `css`?

Reply:
Let's talk about getting #4001 published, then we can add it here!

Reply:
Ok, that is awesome. I don't remember seeing that earlier. Let me take a look at that and see how we can move it forward. I've got another meeting coming up with someone from another team who is interested in working with us on a VS Code plugin. Something that would generate files and default component setup. Maybe we can get this PR in and publish and then use this plugin as a vehicle to deliver further ergonomics for FAST through tools.